AddonArmTemplate/FullAzureMonitorMetricsProfile.json

```diff
@@ -63,6 +63,11 @@
     "kubernetesRecordingRuleGroup": "KubernetesReccordingRulesRuleGroup-",
     "kubernetesRecordingRuleGroupName": "[concat(variables('kubernetesRecordingRuleGroup'), variables('clusterName'))]",
     "kubernetesRecordingRuleGroupDescription": "Kubernetes Recording Rules RuleGroup",
+    "kubernetesRecordingRuleGroupWin": "KubernetesRecordingRulesRuleGroup-Win-",
```

Comment:
kaveesh, aren't we pointing to this template from our public docs ? If so, after this is merged, any newer downloads will pick up these rules , which we dont want.

Reply:
remove this for now. Add a task to merge this in when releasing 3p addon

Reply:
https://github.com/Azure/prometheus-collector/pull/366